jupyter-pytorch-cuda-full/rockcraft.yaml

```diff
@@ -0,0 +1,185 @@
+name: jupyter-pytorch-cuda-full
+summary: An image for using Jupyter & PyTorch on CUDA
+description: |
+  This image is used as part of the Charmed Kubeflow product. It is deployed
+  in a user namespace, and enables users to perform CUDA-accelerated tasks
+  using PyTorch from within a Jupyter Notebook.
+
+  Both PyTorch and Jupyter are installed in Conda environment, which is
+  automatically activated.
+version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  jupyter:
+    override: replace
+    summary: "jupyter-pytorch-cuda-full service"
+    startup: enabled
+    user: jovyan
+    environment:
+      NB_USER: jovyan
+      NB_UID: 1001
+      NB_PREFIX: /
+      HOME: /home/jovyan
+      SHELL: /bin/bash
+      LANG: en_US.UTF-8
+      LANGUAGE: en_US.UTF-8
+      LC_ALL: en_US.UTF-8
+      PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      NVIDIA_VISIBLE_DEVICES: all
+      NVIDIA_DRIVER_CAPABILITIES: compute,utility
+      LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64
+    command: /opt/conda/bin/jupyter lab --notebook-dir="/home/jovyan" --ip=0.0.0.0 --no-browser --allow-root --port=8888 --ServerApp.token="" --ServerApp.password="" --ServerApp.allow_origin="*" --ServerApp.base_url="/" --ServerApp.authenticate_prometheus=False
+platforms:
+  amd64:
+
+parts:
+  overlay-pkgs:
```

Comment:
Is this part even useful ? alll these packages are installed but nothing happens in this part.

Reply:
Looks like those packages will be available for other parts:
https://canonical-craft-parts.readthedocs-hosted.com/en/latest/explanation/overlays.html#overlay-parameters